test/strategy/yearn/YearnStrategy.spec.ts

```diff
@@ -145,6 +145,36 @@ describe('YearnStrategy', () => {
     });
   });
 
+  describe('#transferOwnership', () => {
+    it('can only be called by the current owner', async () => {
+      await expect(
+        strategy.connect(alice).transferOwnership(alice.address),
+      ).to.be.revertedWith('Ownable: caller is not the owner');
+    });
+
+    it('reverts if new owner is address(0)', async () => {
+      await expect(
+        strategy.connect(owner).transferOwnership(constants.AddressZero),
+      ).to.be.revertedWith('StrategyOwnerCannotBe0Address');
+    });
+
+    it('changes ownership to the new owner', async () => {
+      await strategy.connect(owner).transferOwnership(alice.address);
+
+      expect(await strategy.owner()).to.be.equal(alice.address);
+    });
+
+    it("revokes previous owner's ADMIN role and sets up ADMIN role for the new owner", async () => {
```

Comment:
this is actually formatted like this by prettier. Because of the single quote in `owner's`, if I change to single quotes and use `owner\'s`, prettier will reformat it back to as it is right now

Reply:
Gotcha, I've seen that before. Missed it here!

Resolved